### ISSUE TITLE
kontena/lb fails healthchecks when deployed without any linked services

### ISSUE THREAD

**Issue by SpComb** [kontena/kontena#3121]
Installing a new `kontena/lb` service with healthchecks will fail:

```
$ kontena stack install terom/lb
> Custom port mappings (comma separated, eg 2022:2022,25:25) : 
 [done] Creating stack lb      
 [done] Triggering deployment of stack lb     
 [done] Waiting for deployment to start     
 [fail] Deploying service lb     
Deployment of service test/lb/lb failed:
  - halting deploy of test/lb/lb, one or more instances failed
  - GridServiceInstanceDeployer::ServiceError: RuntimeError: container restarted (on node terom-test-edge-1)
```

This is because the new `haproxy.cfg` will no longer contain any `bind :80` if there are no linked services: https://github.com/kontena/kontena-loadbalancer/blob/006b58a7bb2ebd7403a141abb5cc921db145e130/lib/kontena/templates/haproxy/main.text.erb#L32-L34

I don't think this new behavior is ideal, because it will break new LB deploys.

**Comment by SpComb** [comment]
Fixed in `kontena/lb:1.1.1` per https://github.com/kontena/kontena-loadbalancer/pull/41 

The LB will now always listen on port 80 with `monitor-uri` if configured with `KONTENA_LB_HEALTH_URI=...`.

**Comment by SpComb** [comment]
Note that this issue still applies to other non-`kontena/ingress-lb` stacks using the new `kontena/lb:1.1` without configuring `KONTENA_LB_HEALTH_URI=...`. They will fail their initial deployment if using `wait_for_port` and/or healthchecks on port 80. That behavior has changed from the classic `kontena/lb:old`, which would always listen on port 80.
